status-models/src/main/java/com/telecominfraproject/wlan/status/equipment/models/EquipmentChannelStatusData.java

```diff
@@ -0,0 +1,79 @@
+package com.telecominfraproject.wlan.status.equipment.models;
+
+import java.util.EnumMap;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Map.Entry;
+
+import com.telecominfraproject.wlan.core.model.equipment.RadioType;
+import com.telecominfraproject.wlan.status.models.StatusDataType;
+import com.telecominfraproject.wlan.status.models.StatusDetails;
+
+public class EquipmentChannelStatusData extends StatusDetails {
+	private static final long serialVersionUID = 8570180206588083190L;
+
+    private Map<RadioType, Integer> channelStatusDataMap = new EnumMap<>(RadioType.class);
```

Comment:
This keeps track of currently active channel per radio, right? 
Would it make sense to also track the current backup channel per radio as used/reported by AP? 
I'm not saying that it's necessary, just asking if it makes sense.

Reply:
The backup channel is on rrm, currently AP only has support Wifi_RRM_Config, there is no State table yet.